src/components/Header/Header.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBell, faSearch } from '@fortawesome/free-solid-svg-icons';
+import './Header.scss';
+import siteLogo from '../../assets/images/logo_ah_secondo.png';
+import siteLogoSmall from '../../assets/images/logo_ah_small.png';
+import { Img, Button } from '../common';
+import HeaderUserMenu from './HeaderUserMenu/HeaderUserMenu';
+import HeaderUserImage from './HeaderUserImage/HeaderUserImage';
+
+class Header extends Component {
+  state = { showUserMenu: false };
+
+  componentDidMount = () => {
+    window.document.addEventListener('click', (e) => {
```

Comment:
I think i am not understanding this part, can you help me?

Reply:
This is useful for a user to hide the menu by just clicking somewhere in the window

Reply:
Thank you.